e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -111,6 +175,28 @@ describe('at-mention', () => {
         cy.getCurrentChannelId().then((id) => {
             offTopicChannelId = id;
         });
+
+        // Set EnableOpenServer to true so users can sign up
+        const newSettings = {
+            TeamSettings: {EnableOpenServer: true},
+        };
+        cy.apiUpdateConfig(newSettings);
+    });
+
+    it('MM7881 Disable default mention notification settings for "case sensitive first name" and "non-case sensitive username"', () => {
+        ignoreUncaughtException();
+
+        // # Signup a new user with an email address and user generated in signupWithEmail
+        signupWithEmailCreateTeam('unique.' + uniqueUserId, 'unique1pw', 'team.' + uniqueTeamId);
+
```

Comment:
Hi @Ths2-9Y-LqJt6 
Currently the testcase fails because it has other authentication methods enabled in the system. Instead of creating a user via UI, we have an API command which you can easily use instead. 

```
cy.loginAsNewUser();
```